odbc_interface.c

```diff
@@ -2087,6 +2104,8 @@ SQLForeignKeys (SQLHSTMT StatementHandle,
 
   SQLCHAR *stPKTableName = NULL;
   SQLCHAR *stFKTableName = NULL;
+  char buf_p[512], buf_f [512];
```

Comment:
It appears to be an unused variable. (buf_p, buf_f)

Reply:
Thanks for the careful review!!!
Fixed.